addons/website_sale/views/templates.xml

```diff
@@ -800,15 +800,17 @@
                     <t t-foreach="entries" t-as="c" t-if="c.image_128" t-set="atLeastOneImage" t-value="True"/>
                     <t t-if="category.parent_id" t-set="backUrl" t-value="keep('/shop/category/' + slug(category.parent_id), category=0)"/>
                     <t t-else="" t-set="backUrl" t-value="'/shop'"/>
-
+                    <!-- TODO VCR don't show sibling category on last nested level -->
```

Comment:
If it's confirmed that it will be done after the release could you remove the note?